### ISSUE TITLE
'type' object is not subscriptable

### ISSUE THREAD

**Issue by mymy5502** [bingganhe123/60s-#3]
![image](https://user-images.githubusercontent.com/60425129/155325490-864ef4a4-d074-458d-a5b2-267854542e9a.png)

**Comment by bingganhe123** [comment]
抱歉，这个问题我也解决不了，也有一个人反馈了和你同样的问题，我以为是我电脑的问题，后来重新装了一台主机安装也没有遇到相同的情况，这个真的无能为力，对不起，是我太菜了

**Comment by gitkeen** [comment]
@bingganhe123 你的python是不是3.9的？

**Comment by bingganhe123** [comment]
是的![20220311124724(1).png](https://user-images.githubusercontent.com/98812723/157803930-b75ba5c5-ea71-4830-a1d6-d8f743d13785.png)

**Comment by bingganhe123** [comment]
> @bingganhe123 你的python是不是3.9的？

![20220311124724(1).png](https://user-images.githubusercontent.com/98812723/157803962-c8545602-9e9f-4249-86d6-e1d5a8272a05.png)

**Comment by gitkeen** [comment]
> 

Python 3.7.12

**Comment by gitkeen** [comment]
> > @bingganhe123 你的python是不是3.9的？
> 
> ![20220311124724(1).png](https://user-images.githubusercontent.com/98812723/157803962-c8545602-9e9f-4249-86d6-e1d5a8272a05.png)

Python 3.7.12
谷歌了一下应该是版本的问题

**Comment by bingganhe123** [comment]
> > > @bingganhe123 你的python是不是3.9的？
> > 
> > ![20220311124724(1).png](https://user-images.githubusercontent.com/98812723/157803962-c8545602-9e9f-4249-86d6-e1d5a8272a05.png)
> 
> Python 3.7.12
> 谷歌了一下应该是版本的问题

有可能是这个问题，我回去试一下，感谢您的帮助

**Comment by gitkeen** [comment]
> > > > @bingganhe123 你的python是不是3.9的？
> > > 
> > > 
> > > ![20220311124724(1).png](https://user-images.githubusercontent.com/98812723/157803962-c8545602-9e9f-4249-86d6-e1d5a8272a05.png)
> > 
> > 
> > Python 3.7.12
> > 谷歌了一下应该是版本的问题
> 
> 有可能是这个问题，我回去试一下，感谢您的帮助

换了python3.9的环境成功运行

**Comment by bingganhe123** [comment]
> > > > > @bingganhe123 你的python是不是3.9的？
> > > > 
> > > > 
> > > > ![20220311124724(1).png](https://user-images.githubusercontent.com/98812723/157803962-c8545602-9e9f-4249-86d6-e1d5a8272a05.png)
> > > 
> > > 
> > > Python 3.7.12
> > > 谷歌了一下应该是版本的问题
> > 
> > 有可能是这个问题，我回去试一下，感谢您的帮助
> 
> 换了python3.9的环境成功运行

我在新的readme和setup中做了说明并打包上传了新版本，感谢您的帮助

**Comment by AvataGarde** [comment]
3.8方法 
在config.py里添加
from typing import List

config class 里改成
read_qq_friends: List[int]
read_qq_groups: List[int]
read_inform_time: List[Time] = []

**Comment by 15x3** [comment]
> 3.8方法 在config.py里添加 from typing import List
> 
> config class 里改成 read_qq_friends: List[int] read_qq_groups: List[int] read_inform_time: List[Time] = []

感谢，经测试py3.10同样可用
